src/NHPersistenceCompatibilityTests/Common/NugetHelper.cs

```diff
@@ -0,0 +1,54 @@
+﻿using NuGet;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+
+namespace Common
+{
+    public class NugetHelper
+    {
+        string packageSource;
+        string fallbackPackageSource;
+
+        public NugetHelper(string packageSource = "https://packages.nuget.org/api/v2", string fallbackPackageSource = "https://www.myget.org/F/particular/")
```

Comment:
What is the reason these values are in code? Also, next time we add a third argument?